packages/magic_sdk/lib/modules/base_module.dart

```diff
@@ -29,4 +29,20 @@ class BaseModule {
     return provider.send(
         request: request, completer: Completer<JavaScriptMessage>());
   }
+
+  Future<JavaScriptMessage> sendToProviderWithList({
```

Comment:
Curious, `sendToProvider` and `sendToProviderWithList` seem pretty identical. What's the use case for this new function 🤔

Reply:
I kept getting  the following error when I used `sendToProvider` as is:  🤔 

```
Unhandled Exception: {code: -32603, message: Internal error: object is not iterable (cannot read property Symbol(Symbol.iterator))}
```

Reply:
I see what you mean though, one sec, I'll try it again 😅